Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3619 @@
+{
```

Comment:
<p>I tried running the notebook and it failed here with an invalid syntax error:</p><p> </p><p><strong>File "&lt;ipython-input-7-402f9dfacd52&gt;", line 1</strong></p><p><strong>    df_within_range['end_date'] = (df_within_range['end_date'], utc=True)</strong></p><p><strong>                                                                   ^</strong></p><p><strong>SyntaxError:</strong> invalid syntax</p><p> </p><p>Perhaps this is a Python 2 vs Python 3 issue?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Looks like it's missing the function name: <code>pd.to_datetime(df_within_range['end_date'], utc=True)</code></p><p>?</p>

Reply:
Yes. Fixed